<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Bibliography | Portfolio Data Science for Biology</title>
  <meta name="description" content="Bibliography | Portfolio Data Science for Biology" />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="Bibliography | Portfolio Data Science for Biology" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="philinevbekkum/dsfb2_workflows_portfolio" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Bibliography | Portfolio Data Science for Biology" />
  
  
  

<meta name="author" content="Philine van Bekkum" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="parameterized-report-of-covid-19-data-from-ecdc.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.4/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="toc"><h2>Portfolio</h2></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html"><i class="fa fa-check"></i>Identification of Antibiotic Resistance Genes in Bacteria</a>
<ul>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#amr-package-in-r"><i class="fa fa-check"></i>AMR package in R</a>
<ul>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#tidy-data"><i class="fa fa-check"></i>Tidy data</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#tidy-data-per-bacteria"><i class="fa fa-check"></i>Tidy data per bacteria</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#combined-tidy-data-of-every-bacteria"><i class="fa fa-check"></i>Combined tidy data of every bacteria</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#wide-data-for-amr-package-functions"><i class="fa fa-check"></i>Wide data for AMR package functions</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#antibiogram"><i class="fa fa-check"></i>Antibiogram</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#quality-control-with-nanoplot-and-nanofilt"><i class="fa fa-check"></i>Quality control with NanoPlot and NanoFilt</a>
<ul>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#fastqc-report-per-base-sequence-quality"><i class="fa fa-check"></i>FastQC Report: per base sequence quality</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#fastqc-report-per-sequence-quality-scores"><i class="fa fa-check"></i>FastQC Report: per sequence quality scores</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-weighted-histogram-of-read-lengths"><i class="fa fa-check"></i>NanoPlot: Weighted histogram of read lengths</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-weighted-histogram-of-read-lengths-after-log-transformation"><i class="fa fa-check"></i>NanoPlot: Weighted histogram of read lengths after log transformation</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-non-weighted-histogram-of-read-lengths"><i class="fa fa-check"></i>NanoPlot: Non weighted histogram of read lengths</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-non-weighted-histogram-of-read-lengths-after-log-transformation"><i class="fa fa-check"></i>NanoPlot: Non weighted histogram of read lengths after log transformation</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-yield-by-length"><i class="fa fa-check"></i>NanoPlot: Yield by length</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-read-lengths-vs-average-read-quality-kde-plot"><i class="fa fa-check"></i>Nanoplot: Read lengths vs Average read quality kde plot</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-read-lengths-vs-average-read-quality-using-dots"><i class="fa fa-check"></i>NanoPlot: Read lengths vs Average read quality using dots</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html"><i class="fa fa-check"></i>Guerilla analytics</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html"><i class="fa fa-check"></i><em>C. elegans</em> Plate Experiment</a>
<ul>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#set-up"><i class="fa fa-check"></i>Set up</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#importing-the-data"><i class="fa fa-check"></i>Importing the data</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#inspecting-the-data-types"><i class="fa fa-check"></i>Inspecting the data types</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#scatterplot-of-c.-elegans-plate-experiment"><i class="fa fa-check"></i>Scatterplot of <em>C. elegans</em> plate experiment</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#normalizing-the-data-for-the-negative-control"><i class="fa fa-check"></i>Normalizing the data for the negative control</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#scatterplot-of-c.-elegans-plate-experiment-with-normalized-data"><i class="fa fa-check"></i>Scatterplot of <em>C. elegans</em> plate experiment with normalized data</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#analyzing-the-c.-elegans-experiment"><i class="fa fa-check"></i>Analyzing the <em>C. elegans</em> experiment</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html"><i class="fa fa-check"></i>Open Peer Review</a>
<ul>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#evaluation-of-transparency-and-reproducibility"><i class="fa fa-check"></i>Evaluation of Transparency and Reproducibility</a>
<ul>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#general-aim"><i class="fa fa-check"></i>General Aim</a></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#results"><i class="fa fa-check"></i>Results</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#evaluation-of-code-quality-and-reproducibility"><i class="fa fa-check"></i>Evaluation of Code Quality and Reproducibility</a>
<ul>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#business-by-country-by-income"><i class="fa fa-check"></i>Business by country by income</a></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#vaccine-by-country-by-education"><i class="fa fa-check"></i>Vaccine by country by education</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html"><i class="fa fa-check"></i>Creating R Packages</a>
<ul>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#developing-the-hittersanalytics-package"><i class="fa fa-check"></i>Developing the HittersAnalytics Package</a></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#installing-the-hittersanalytics-package"><i class="fa fa-check"></i>Installing the HittersAnalytics Package</a></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#examples-of-the-functions-in-the-hittersanalytics-package"><i class="fa fa-check"></i>Examples of the functions in the HittersAnalytics Package</a></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="project-hack-the-vitek.html"><a href="project-hack-the-vitek.html"><i class="fa fa-check"></i>Project “Hack the VITEK”</a>
<ul>
<li class="chapter" data-level="" data-path="project-hack-the-vitek.html"><a href="project-hack-the-vitek.html#vitek2-compact"><i class="fa fa-check"></i>VITEK®2 Compact</a></li>
<li class="chapter" data-level="" data-path="project-hack-the-vitek.html"><a href="project-hack-the-vitek.html#hack-the-vitek"><i class="fa fa-check"></i>Hack the VITEK</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html"><i class="fa fa-check"></i>Parameterized report of COVID-19 data from ECDC</a>
<ul>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html#set-up-1"><i class="fa fa-check"></i>Set up</a></li>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html#load-and-filter-the-covid-19-dataset"><i class="fa fa-check"></i>Load and filter the COVID-19 dataset</a></li>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html#plots-of-the-covid-19-dataset"><i class="fa fa-check"></i>Plots of the COVID-19 dataset</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="bibliography.html"><a href="bibliography.html"><i class="fa fa-check"></i>Bibliography</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Portfolio Data Science for Biology</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="bibliography" class="section level1 unnumbered">
<h1>Bibliography</h1>

<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div class="csl-entry">
<span>“Antimicrobial Resistance Data.”</span> n.d. https://www.kaggle.com/datasets/amritpal333/antimicrobial-resistance-data. Accessed January 3, 2025.
</div>
<div class="csl-entry">
Coster, Wouter De. 2024a. <span>“Wdecoster/Nanofilt.”</span>
</div>
<div class="csl-entry">
———. 2024b. <span>“Wdecoster/<span>NanoPlot</span>.”</span>
</div>
<div class="csl-entry">
<span>“Data on the Daily Number of New Reported <span>COVID-19</span> Cases and Deaths by <span>EU</span>/<span>EEA</span> Country.”</span> 2022. https://www.ecdc.europa.eu/en/publications-data/data-daily-new-cases-covid-19-eueea-country.
</div>
<div class="csl-entry">
<span>“<span>ERR10367339</span> : <span>Run Browser</span> : <span>SRA Archive</span> : <span>NCBI</span>.”</span> n.d. https://trace.ncbi.nlm.nih.gov/Traces/index.html?view=run_browser&amp;acc=ERR10367339&amp;display=metadata. Accessed January 3, 2025.
</div>
<div class="csl-entry">
<span>“How to Conduct <span>AMR</span> Data Analysis.”</span> n.d. https://msberends.github.io/AMR/articles/AMR.html#availability-of-species. Accessed January 2, 2025.
</div>
<div class="csl-entry">
Humphries, Romney, Shelley Campeau, Thomas E. Davis, Kristin J. Nagaro, Vincent J. LaBombardi, Simone Franklin, Lisa Heimbach, and Hari P. Dwivedi. 2021. <span>“Multicenter <span>Evaluation</span> of <span>Ceftazidime-Avibactam Susceptibility Testing</span> of <span><em>Enterobacterales</em></span> and <span>Pseudomonas</span> Aeruginosa on the <span>Vitek</span> 2 <span>System</span>.”</span> Edited by Patricia J. Simner. <em>Journal of Clinical Microbiology</em> 59 (3): e01870–20. <a href="https://doi.org/10.1128/JCM.01870-20">https://doi.org/10.1128/JCM.01870-20</a>.
</div>
<div class="csl-entry">
Jia, Wei, Gang Li, and Wen Wang. 2014. <span>“Prevalence and <span>Antimicrobial Resistance</span> of <span>Enterococcus Species</span>: <span>A Hospital-Based Study</span> in <span>China</span>.”</span> <em>International Journal of Environmental Research and Public Health</em> 11 (3): 3424–42. <a href="https://doi.org/10.3390/ijerph110303424">https://doi.org/10.3390/ijerph110303424</a>.
</div>
<div class="csl-entry">
Lazarus, Jeffrey V., Scott C. Ratzan, Adam Palayew, Lawrence O. Gostin, Heidi J. Larson, Kenneth Rabin, Spencer Kimball, and Ayman El-Mohandes. 2021. <span>“A Global Survey of Potential Acceptance of a <span>COVID-19</span> Vaccine.”</span> <em>Nature Medicine</em> 27 (2): 225–28. <a href="https://doi.org/10.1038/s41591-020-1124-9">https://doi.org/10.1038/s41591-020-1124-9</a>.
</div>
<div class="csl-entry">
Ligozzi, Marco, Cinzia Bernini, Maria Grazia Bonora, Maria De Fatima, Jessica Zuliani, and Roberta Fontana. 2002. <span>“Evaluation of the <span>VITEK</span> 2 <span>System</span> for <span>Identification</span> and <span>Antimicrobial Susceptibility Testing</span> of <span>Medically Relevant Gram-Positive Cocci</span>.”</span> <em>Journal of Clinical Microbiology</em> 40 (5): 1681–86. <a href="https://doi.org/10.1128/JCM.40.5.1681-1686.2002">https://doi.org/10.1128/JCM.40.5.1681-1686.2002</a>.
</div>
<div class="csl-entry">
<span>“<span>OSF</span> <span></span> <span>COVID-19 Vaccine</span> Paper.”</span> n.d. https://osf.io/kzq69/. Accessed January 3, 2025.
</div>
<div class="csl-entry">
Pincus, David H. n.d. <span>“<span>MICROBIAL IDENTIFICATION USING THE BIOM<span>É</span>RIEUX VITEK</span><span></span> 2 <span>SYSTEM</span>.”</span>
</div>
<div class="csl-entry">
<span>“Reproducibility and Reporting Practices in <span>COVID-19</span> Preprint Manuscripts.”</span> n.d. <em>CoLab</em>. https://colab.ws/articles/10.1101%2F2020.03.24.20042796. Accessed January 3, 2025.
</div>
<div class="csl-entry">
Saha, Priyanka, Arup Kumar Banerjee, Prem Prakash Tripathi, Amit Kumar Srivastava, and Upasana Ray. 2020. <span>“A Virus That Has Gone Viral: Amino Acid Mutation in <span>S</span> Protein of <span>Indian</span> Isolate of <span>Coronavirus COVID-19</span> Might Impact Receptor Binding, and Thus, Infectivity.”</span> <em>Bioscience Reports</em> 40 (5): BSR20201312. <a href="https://doi.org/10.1042/BSR20201312">https://doi.org/10.1042/BSR20201312</a>.
</div>
<div class="csl-entry">
Wallet, Frédéric, Caroline Loïez, Emilie Renaux, Nadine Lemaitre, and René J. Courcol. 2005. <span>“Performances of <span>VITEK</span> 2 <span>Colorimetric Cards</span> for <span>Identification</span> of <span>Gram-Positive</span> and <span>Gram-Negative Bacteria</span>.”</span> <em>Journal of Clinical Microbiology</em> 43 (9): 4402–6. <a href="https://doi.org/10.1128/JCM.43.9.4402-4406.2005">https://doi.org/10.1128/JCM.43.9.4402-4406.2005</a>.
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="parameterized-report-of-covid-19-data-from-ecdc.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true,
"toc_depth": 3
}
});
});
</script>

</body>

</html>
