<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Open Peer Review | Portfolio Data Science for Biology</title>
  <meta name="description" content="Open Peer Review | Portfolio Data Science for Biology" />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="Open Peer Review | Portfolio Data Science for Biology" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="philinevbekkum/dsfb2_workflows_portfolio" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Open Peer Review | Portfolio Data Science for Biology" />
  
  
  

<meta name="author" content="Philine van Bekkum" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="c.-elegans-plate-experiment.html"/>
<link rel="next" href="references-1.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.4/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="toc"><h2>Portfolio</h2></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html"><i class="fa fa-check"></i>Identification of Antibiotic Resistance Genes in Bacteria</a>
<ul>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#amr-package-in-r"><i class="fa fa-check"></i>AMR package in R</a>
<ul>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#tidy-data"><i class="fa fa-check"></i>Tidy data</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#tidy-data-per-bacteria"><i class="fa fa-check"></i>Tidy data per bacteria</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#combined-tidy-data-of-every-bacteria"><i class="fa fa-check"></i>Combined tidy data of every bacteria</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#wide-data-for-amr-package-functions"><i class="fa fa-check"></i>Wide data for AMR package functions</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#antibiogram"><i class="fa fa-check"></i>Antibiogram</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#quality-control-with-nanoplot-and-nanofilt"><i class="fa fa-check"></i>Quality control with NanoPlot and NanoFilt</a>
<ul>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#fastqc-report-per-base-sequence-quality"><i class="fa fa-check"></i>FastQC Report: per base sequence quality</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#fastqc-report-per-sequence-quality-scores"><i class="fa fa-check"></i>FastQC Report: per sequence quality scores</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-weighted-histogram-of-read-lengths"><i class="fa fa-check"></i>NanoPlot: Weighted histogram of read lengths</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-weighted-histogram-of-read-lengths-after-log-transformation"><i class="fa fa-check"></i>NanoPlot: Weighted histogram of read lengths after log transformation</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-non-weighted-histogram-of-read-lengths"><i class="fa fa-check"></i>NanoPlot: Non weighted histogram of read lengths</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-non-weighted-histogram-of-read-lengths-after-log-transformation"><i class="fa fa-check"></i>NanoPlot: Non weighted histogram of read lengths after log transformation</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-yield-by-length"><i class="fa fa-check"></i>NanoPlot: Yield by length</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-read-lengths-vs-average-read-quality-kde-plot"><i class="fa fa-check"></i>Nanoplot: Read lengths vs Average read quality kde plot</a></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#nanoplot-read-lengths-vs-average-read-quality-using-dots"><i class="fa fa-check"></i>NanoPlot: Read lengths vs Average read quality using dots</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="identification-of-antibiotic-resistance-genes-in-bacteria.html"><a href="identification-of-antibiotic-resistance-genes-in-bacteria.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html"><i class="fa fa-check"></i>Guerilla analytics</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html"><i class="fa fa-check"></i><em>C. elegans</em> Plate Experiment</a>
<ul>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#set-up"><i class="fa fa-check"></i>Set up</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#importing-the-data"><i class="fa fa-check"></i>Importing the data</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#inspecting-the-data-types"><i class="fa fa-check"></i>Inspecting the data types</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#scatterplot-of-c.-elegans-plate-experiment"><i class="fa fa-check"></i>Scatterplot of <em>C. elegans</em> plate experiment</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#normalizing-the-data-for-the-negative-control"><i class="fa fa-check"></i>Normalizing the data for the negative control</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#scatterplot-of-c.-elegans-plate-experiment-with-normalized-data"><i class="fa fa-check"></i>Scatterplot of <em>C. elegans</em> plate experiment with normalized data</a></li>
<li class="chapter" data-level="" data-path="c.-elegans-plate-experiment.html"><a href="c.-elegans-plate-experiment.html#analyzing-the-c.-elegans-experiment"><i class="fa fa-check"></i>Analyzing the <em>C. elegans</em> experiment</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html"><i class="fa fa-check"></i>Open Peer Review</a>
<ul>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#evaluation-of-transparency-and-reproducibility"><i class="fa fa-check"></i>Evaluation of Transparency and Reproducibility</a>
<ul>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#general-aim"><i class="fa fa-check"></i>General Aim</a></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#results"><i class="fa fa-check"></i>Results</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#evaluation-of-code-quality-and-reproducibility"><i class="fa fa-check"></i>Evaluation of Code Quality and Reproducibility</a>
<ul>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#business-by-country-by-income"><i class="fa fa-check"></i>Business by country by income</a></li>
<li class="chapter" data-level="" data-path="open-peer-review.html"><a href="open-peer-review.html#vaccine-by-country-by-education"><i class="fa fa-check"></i>Vaccine by country by education</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html"><i class="fa fa-check"></i>Creating R Packages</a>
<ul>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#developing-the-hittersanalytics-package"><i class="fa fa-check"></i>Developing the HittersAnalytics Package</a></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#installing-the-hittersanalytics-package"><i class="fa fa-check"></i>Installing the HittersAnalytics Package</a></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#examples-of-the-functions-in-the-hittersanalytics-package"><i class="fa fa-check"></i>Examples of the functions in the HittersAnalytics Package</a></li>
<li class="chapter" data-level="" data-path="creating-r-packages.html"><a href="creating-r-packages.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="project-hack-the-vitek.html"><a href="project-hack-the-vitek.html"><i class="fa fa-check"></i>Project “Hack the VITEK”</a>
<ul>
<li class="chapter" data-level="" data-path="project-hack-the-vitek.html"><a href="project-hack-the-vitek.html#vitek2-compact"><i class="fa fa-check"></i>VITEK®2 Compact</a></li>
<li class="chapter" data-level="" data-path="project-hack-the-vitek.html"><a href="project-hack-the-vitek.html#hack-the-vitek"><i class="fa fa-check"></i>Hack the VITEK</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html"><i class="fa fa-check"></i>Parameterized report of COVID-19 data from ECDC</a>
<ul>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html#set-up-1"><i class="fa fa-check"></i>Set up</a></li>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html#load-and-filter-the-covid-19-dataset"><i class="fa fa-check"></i>Load and filter the COVID-19 dataset</a></li>
<li class="chapter" data-level="" data-path="parameterized-report-of-covid-19-data-from-ecdc.html"><a href="parameterized-report-of-covid-19-data-from-ecdc.html#plots-of-the-covid-19-dataset"><i class="fa fa-check"></i>Plots of the COVID-19 dataset</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="bibliography.html"><a href="bibliography.html"><i class="fa fa-check"></i>Bibliography</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Portfolio Data Science for Biology</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="open-peer-review" class="section level1 unnumbered">
<h1>Open Peer Review</h1>
<div id="evaluation-of-transparency-and-reproducibility" class="section level2">
<h2>Evaluation of Transparency and Reproducibility</h2>
<p class="justify">
The selected article, “A virus that has gone viral: Amino acid mutation in S protein of Indian isolate of Coronavirus COVID-19 might impact receptor binding and thus infectivity”, examines amino acid mutations in the spike protein of SARS-CoV-2 isolates from India and their potential impact on receptor binding and infectivity <span class="citation">(Saha et al. 2020)</span>.
</p>
<p class="justify">
The transparency of the selected study was evaluated based on predefined criteria for reproducibility and reporting practices, as described by Summer et al. (2020) <span class="citation">(<span>“Reproducibility and Reporting Practices in <span>COVID-19</span> Preprint Manuscripts”</span> n.d.)</span> in their work on reproducibility and reporting practices in COVID-19 preprint manuscripts. The table below summaries the findings.
</p>
<p><em>Table 1. Transparency Criteria</em></p>
<table>
<colgroup>
<col width="40%" />
<col width="59%" />
</colgroup>
<thead>
<tr class="header">
<th>Transparency Criteria</th>
<th>Response Type</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Study Purpose</td>
<td>TRUE</td>
</tr>
<tr class="even">
<td>Data Availability Statement</td>
<td>FALSE</td>
</tr>
<tr class="odd">
<td>Data Location</td>
<td>GenBank</td>
</tr>
<tr class="even">
<td>Study Location</td>
<td>TRUE; India</td>
</tr>
<tr class="odd">
<td>Author Review</td>
<td>Professional; Contact information is available</td>
</tr>
<tr class="even">
<td>Ethics Statement</td>
<td>FALSE</td>
</tr>
<tr class="odd">
<td>Funding Statement</td>
<td>TRUE</td>
</tr>
<tr class="even">
<td>Code Availability</td>
<td>FALSE</td>
</tr>
</tbody>
</table>
<div id="general-aim" class="section level3">
<h3>General Aim</h3>
<p class="justify">
The general aim of this article was to analyze and compare mutations in the spike protein Indian SARS-CoV-2 isolates compared to Wuhan and Italian isolates, to identify notable features, especially mutations that could affect viral infectivity.
</p>
</div>
<div id="methods" class="section level3">
<h3>Methods</h3>
<p class="justify">
The researchers obtained complete spike protein sequences of SARS-CoV-2 from GenBank, including sequences 17 sequences from Wuhan, one from Italy, and two from India (isolates 29 and 166). Sequence alignments were performed using NCBI BLAST, CLUSTAL W, and CLUSTAL OMEGA. Secondary structure prediction was done using CFSSP server, and mutation analysis was done using Mutprep server. JMol and ConSurf tools were used to predict protein structure. Further kinetics and structure analyses were performed using Dynemut Server and Chimera version 11.
</p>
</div>
<div id="results" class="section level3">
<h3>Results</h3>
<p class="justify">
All 17 sequences from Wuhan were identical and matched completely with the sequence from Italy. However two Indian isolates (29 en 166) showed notable mutations. Isolate 19 had a deletion mutation of tyrosine at position 144 and a substitution mutation from arginine to isoleucine at position 407, affecting the receptor-binding domain. Isolate 166 had a substitution mutation from alanine to valine at position 930 in the S2 subunits of the spike protein. These mutations could potentially alter the viral entry and infectivity, with implications for viral evolution and drug target design. Further studies are required to validate these findings and understand their impact on virus behavior.
</p>
</div>
</div>
<div id="evaluation-of-code-quality-and-reproducibility" class="section level2">
<h2>Evaluation of Code Quality and Reproducibility</h2>
<p class="justify">
The selected article, “A global survey of potential acceptance of a COVID-19 vaccine”, examines global COVID-19 vaccine acceptance rates and factors influencing willingness to vaccinate, based on a survey of 13,426 people across 19 countries <span class="citation">(Lazarus et al. 2021)</span>.
</p>
<p class="justify">
The quality of the code from the selected study was examined by downloading the code from the OSF website and executing the code in R. The R code is designed to evaluate survey data on attitudes toward COVID-19 vaccination. The code includes data preprocessing, visualization, and statistic modeling. The code starts with loading a CSV file with the data and sets up various demographic and survey response variables, like gender, income, education, age group, and country. The data is transformed into categorical variables for better interpretation <span class="citation">(<span>“<span>OSF</span> <span></span> <span>COVID-19 Vaccine</span> Paper”</span> n.d.)</span>.
</p>
<p class="justify">
The analysis covers two main survey question: one about the willingness to follow an employers recommendation and the other about the willingness to take the vaccine after it was proven safe and effective. The responses to these questions are visualized in bar graphs segmented by demographics and country. Logistic regression models are used to show the relationship between demographic factors and the acceptance of the vaccines by country.
</p>
<p class="justify">
To successfully execute the code, I made several adjustments: I modified the file path to the data, I commented out the case factor because it wasn’t necessary to proceed with the code, and I commented out the ggsave() function because this was also not necessary to proceed with the code and the given file path didn’t exist. To save the pdf file it is also possible to change the file path. Lastly, I installed the tableone package because it wasn’t installed yet. Overall, it was easy to reproduce the code. I would score the reproducibility of the visualization of this project at a 4, with 1 being very hard and 5 being very easy.
</p>
<p class="justify">
Below you can find some examples from the R code generating figures “Business by country by income” and “Vaccine by country by education”.
</p>
<div id="business-by-country-by-income" class="section level3">
<h3>Business by country by income</h3>
<div class="sourceCode" id="cb83"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb83-1"><a href="open-peer-review.html#cb83-1" tabindex="-1"></a>data<span class="sc">$</span>Busines2_fct <span class="ot">&lt;-</span> <span class="fu">factor</span>(data<span class="sc">$</span>Business2,</span>
<span id="cb83-2"><a href="open-peer-review.html#cb83-2" tabindex="-1"></a>                            <span class="at">levels =</span> <span class="fu">c</span>(</span>
<span id="cb83-3"><a href="open-peer-review.html#cb83-3" tabindex="-1"></a>                              <span class="dv">1</span>,</span>
<span id="cb83-4"><a href="open-peer-review.html#cb83-4" tabindex="-1"></a>                              <span class="dv">2</span>,</span>
<span id="cb83-5"><a href="open-peer-review.html#cb83-5" tabindex="-1"></a>                              <span class="dv">3</span>,</span>
<span id="cb83-6"><a href="open-peer-review.html#cb83-6" tabindex="-1"></a>                              <span class="dv">4</span>,</span>
<span id="cb83-7"><a href="open-peer-review.html#cb83-7" tabindex="-1"></a>                              <span class="dv">5</span></span>
<span id="cb83-8"><a href="open-peer-review.html#cb83-8" tabindex="-1"></a>                            ),</span>
<span id="cb83-9"><a href="open-peer-review.html#cb83-9" tabindex="-1"></a>                            <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb83-10"><a href="open-peer-review.html#cb83-10" tabindex="-1"></a>                              <span class="st">&quot;Completely disagree&quot;</span>,</span>
<span id="cb83-11"><a href="open-peer-review.html#cb83-11" tabindex="-1"></a>                              <span class="st">&quot;Somewhat disagree&quot;</span>,</span>
<span id="cb83-12"><a href="open-peer-review.html#cb83-12" tabindex="-1"></a>                              <span class="st">&quot;Neutral/no opinion&quot;</span>,</span>
<span id="cb83-13"><a href="open-peer-review.html#cb83-13" tabindex="-1"></a>                              <span class="st">&quot;Somewhat agree&quot;</span>,</span>
<span id="cb83-14"><a href="open-peer-review.html#cb83-14" tabindex="-1"></a>                              <span class="st">&quot;Completely agree&quot;</span></span>
<span id="cb83-15"><a href="open-peer-review.html#cb83-15" tabindex="-1"></a>                            ))</span>
<span id="cb83-16"><a href="open-peer-review.html#cb83-16" tabindex="-1"></a></span>
<span id="cb83-17"><a href="open-peer-review.html#cb83-17" tabindex="-1"></a></span>
<span id="cb83-18"><a href="open-peer-review.html#cb83-18" tabindex="-1"></a><span class="fu">summary</span>(data<span class="sc">$</span>Busines2_fct)</span></code></pre></div>
<pre><code>## Completely disagree   Somewhat disagree  Neutral/no opinion      Somewhat agree 
##                1179                2299                3488                4579 
##    Completely agree 
##                1881</code></pre>
<div class="sourceCode" id="cb85"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb85-1"><a href="open-peer-review.html#cb85-1" tabindex="-1"></a>inc <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(</span>
<span id="cb85-2"><a href="open-peer-review.html#cb85-2" tabindex="-1"></a>  <span class="at">data =</span> data</span>
<span id="cb85-3"><a href="open-peer-review.html#cb85-3" tabindex="-1"></a>)<span class="sc">+</span></span>
<span id="cb85-4"><a href="open-peer-review.html#cb85-4" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(</span>
<span id="cb85-5"><a href="open-peer-review.html#cb85-5" tabindex="-1"></a>    <span class="at">x =</span> country_name,</span>
<span id="cb85-6"><a href="open-peer-review.html#cb85-6" tabindex="-1"></a>    <span class="at">fill =</span> Busines2_fct</span>
<span id="cb85-7"><a href="open-peer-review.html#cb85-7" tabindex="-1"></a>  ), <span class="at">stat =</span> <span class="st">&#39;count&#39;</span>, <span class="at">position =</span> <span class="st">&#39;fill&#39;</span>)<span class="sc">+</span></span>
<span id="cb85-8"><a href="open-peer-review.html#cb85-8" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>ww_fct)<span class="sc">+</span></span>
<span id="cb85-9"><a href="open-peer-review.html#cb85-9" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">size =</span> <span class="dv">10</span>))<span class="sc">+</span></span>
<span id="cb85-10"><a href="open-peer-review.html#cb85-10" tabindex="-1"></a>  <span class="fu">coord_flip</span>()<span class="sc">+</span></span>
<span id="cb85-11"><a href="open-peer-review.html#cb85-11" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">&quot;Business by country by income&quot;</span>)<span class="sc">+</span></span>
<span id="cb85-12"><a href="open-peer-review.html#cb85-12" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Proportion&quot;</span>)<span class="sc">+</span></span>
<span id="cb85-13"><a href="open-peer-review.html#cb85-13" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Country&quot;</span>)<span class="sc">+</span></span>
<span id="cb85-14"><a href="open-peer-review.html#cb85-14" tabindex="-1"></a>  <span class="fu">scale_fill_brewer</span>(<span class="st">&quot;Response&quot;</span>, <span class="at">type =</span> <span class="st">&quot;div&quot;</span>, <span class="at">palette =</span> <span class="dv">4</span>)</span>
<span id="cb85-15"><a href="open-peer-review.html#cb85-15" tabindex="-1"></a>inc</span></code></pre></div>
<p><img src="figures/business%20by%20country%20by%20income-1.png" width="672" /></p>
<p><em>Figuur 1. Business by country by income.</em></p>
</div>
<div id="vaccine-by-country-by-education" class="section level3">
<h3>Vaccine by country by education</h3>
<div class="sourceCode" id="cb86"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb86-1"><a href="open-peer-review.html#cb86-1" tabindex="-1"></a>data<span class="sc">$</span>Vaccine_fct <span class="ot">&lt;-</span> <span class="fu">factor</span>(data<span class="sc">$</span>Vaccine,</span>
<span id="cb86-2"><a href="open-peer-review.html#cb86-2" tabindex="-1"></a>                           <span class="at">levels =</span> <span class="fu">c</span>(</span>
<span id="cb86-3"><a href="open-peer-review.html#cb86-3" tabindex="-1"></a>                             <span class="dv">1</span>,</span>
<span id="cb86-4"><a href="open-peer-review.html#cb86-4" tabindex="-1"></a>                             <span class="dv">2</span>,</span>
<span id="cb86-5"><a href="open-peer-review.html#cb86-5" tabindex="-1"></a>                             <span class="dv">3</span>,</span>
<span id="cb86-6"><a href="open-peer-review.html#cb86-6" tabindex="-1"></a>                             <span class="dv">4</span>,</span>
<span id="cb86-7"><a href="open-peer-review.html#cb86-7" tabindex="-1"></a>                             <span class="dv">5</span></span>
<span id="cb86-8"><a href="open-peer-review.html#cb86-8" tabindex="-1"></a>                           ),</span>
<span id="cb86-9"><a href="open-peer-review.html#cb86-9" tabindex="-1"></a>                           <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb86-10"><a href="open-peer-review.html#cb86-10" tabindex="-1"></a>                             <span class="st">&quot;Completely disagree&quot;</span>,</span>
<span id="cb86-11"><a href="open-peer-review.html#cb86-11" tabindex="-1"></a>                             <span class="st">&quot;Somewhat disagree&quot;</span>,</span>
<span id="cb86-12"><a href="open-peer-review.html#cb86-12" tabindex="-1"></a>                             <span class="st">&quot;Neutral/no opinion&quot;</span>,</span>
<span id="cb86-13"><a href="open-peer-review.html#cb86-13" tabindex="-1"></a>                             <span class="st">&quot;Somewhat agree&quot;</span>,</span>
<span id="cb86-14"><a href="open-peer-review.html#cb86-14" tabindex="-1"></a>                             <span class="st">&quot;Completely agree&quot;</span></span>
<span id="cb86-15"><a href="open-peer-review.html#cb86-15" tabindex="-1"></a>                           ))</span>
<span id="cb86-16"><a href="open-peer-review.html#cb86-16" tabindex="-1"></a></span>
<span id="cb86-17"><a href="open-peer-review.html#cb86-17" tabindex="-1"></a></span>
<span id="cb86-18"><a href="open-peer-review.html#cb86-18" tabindex="-1"></a><span class="fu">summary</span>(data<span class="sc">$</span>Vaccine_fct)</span></code></pre></div>
<pre><code>## Completely disagree   Somewhat disagree  Neutral/no opinion      Somewhat agree 
##                1091                 819                1912                3318 
##    Completely agree 
##                6286</code></pre>
<div class="sourceCode" id="cb88"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb88-1"><a href="open-peer-review.html#cb88-1" tabindex="-1"></a>educ <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(</span>
<span id="cb88-2"><a href="open-peer-review.html#cb88-2" tabindex="-1"></a>  <span class="at">data =</span> data</span>
<span id="cb88-3"><a href="open-peer-review.html#cb88-3" tabindex="-1"></a>)<span class="sc">+</span></span>
<span id="cb88-4"><a href="open-peer-review.html#cb88-4" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(</span>
<span id="cb88-5"><a href="open-peer-review.html#cb88-5" tabindex="-1"></a>    <span class="at">x =</span> country_name,</span>
<span id="cb88-6"><a href="open-peer-review.html#cb88-6" tabindex="-1"></a>    <span class="at">fill =</span> Vaccine_fct</span>
<span id="cb88-7"><a href="open-peer-review.html#cb88-7" tabindex="-1"></a>  ), <span class="at">stat =</span> <span class="st">&#39;count&#39;</span>, <span class="at">position =</span> <span class="st">&#39;fill&#39;</span>)<span class="sc">+</span></span>
<span id="cb88-8"><a href="open-peer-review.html#cb88-8" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>educ_fact)<span class="sc">+</span></span>
<span id="cb88-9"><a href="open-peer-review.html#cb88-9" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">size =</span> <span class="dv">10</span>))<span class="sc">+</span></span>
<span id="cb88-10"><a href="open-peer-review.html#cb88-10" tabindex="-1"></a>  <span class="fu">coord_flip</span>()<span class="sc">+</span></span>
<span id="cb88-11"><a href="open-peer-review.html#cb88-11" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">&quot;Vaccine by country by education&quot;</span>)<span class="sc">+</span></span>
<span id="cb88-12"><a href="open-peer-review.html#cb88-12" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Proportion&quot;</span>)<span class="sc">+</span></span>
<span id="cb88-13"><a href="open-peer-review.html#cb88-13" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Country&quot;</span>)<span class="sc">+</span></span>
<span id="cb88-14"><a href="open-peer-review.html#cb88-14" tabindex="-1"></a>  <span class="fu">scale_fill_brewer</span>(<span class="st">&quot;Response&quot;</span>, <span class="at">type =</span> <span class="st">&quot;div&quot;</span>, <span class="at">palette =</span> <span class="dv">4</span>)</span>
<span id="cb88-15"><a href="open-peer-review.html#cb88-15" tabindex="-1"></a>educ</span></code></pre></div>
<p><img src="figures/vaccine%20by%20country%20by%20education-1.png" width="672" /></p>
<p><em>Figuur 2. Vaccine by country by education.</em></p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="c.-elegans-plate-experiment.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references-1.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true,
"toc_depth": 3
}
});
});
</script>

</body>

</html>
